esp/src/src-react/components/controls/Grid.tsx

```diff
@@ -299,7 +305,32 @@ const FluentStoreGrid: React.FunctionComponent<FluentStoreGridProps> = ({
         columnWidths.set(column.key, newWidth);
     }, [columnWidths]);
 
-    return <div data-is-scrollable="true" style={{ overflow: "auto", height: "100%" }}>
+    /*  Monitor Scroll Events (hack)
+
+        Essentially we are setting the scrollElement of the DetailsList to the div that contains the DetailsList (rather than a scrollable pane host).
+        See:  https://github.com/microsoft/fluentui/blob/55d3a31042e8972ea373841bef616c68e6ab69f9/packages/react/src/components/List/List.tsx#L355-L369
+
+        Note: Not sure if `_onScroll` call is needed, but excluding for now as it seems to work without it and is more performant.
+    */
```

Comment:
didn't this get removed a few weeks back in favor of that data-is-scrollable attribute?